carcara/src/parser/mod.rs

```diff
@@ -1417,6 +1437,22 @@ impl<'a, R: BufRead> Parser<'a, R> {
                 }
                 assert_indexed_op_args_value(&op_args, 0..)?;
             }
+            IndexedOperator::RePower => {
+                assert_num_args(&op_args, 1)?;
+                assert_num_args(&args, 1)?;
+                SortError::assert_eq(&Sort::Int, &op_args[0].sort())?;
+                SortError::assert_eq(&Sort::RegLan, &args[0].raw_sort())?;
+                assert_indexed_op_args_value(&[op_args[0].clone()], 0..)?;
```

Comment:
No need to clone here, you can just pass a reference to `op_args` directly:
```suggestion
                assert_indexed_op_args_value(&op_args, 0..)?;

```